CONTRIBUTING.md

```diff
@@ -43,6 +43,104 @@ To contribute your work you need to...
 4. Push to the Branch (`git push origin newBranchName`)
 5. Open a Pull Request
 
+### Commit Message Format
+
+We have very precise rules over how our Git commit messages must be formatted.
+This format leads to **easier to read commit history**.
+
+Each commit message consists of a **header**, a **body**, and a **footer**.
+
+```
+<header>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+#### Commit Message Header
+
+```
+<type>: <subject line>
+  │            │
+  │            └─⫸ Subject in present tense. Capitalized. No period at the end.
+  │
+  │
+  └─⫸ Commit Type: build|ci|docs|feat|fix|perf|refactor|test|revert|style
+```
+
+The `<type>` and `<subject line>` fields are mandatory.
+
+##### Type
+
+Must be one of the following:
+
+* **build**: Changes that affect the build system 
+* **ci**: Changes to our CI configuration files and scripts 
+* **docs**: Documentation changes
+* **feat**: A new feature
+* **fix**: A bug fix
+* **perf**: A code change that improves performance
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **test**: Adding missing tests or correcting existing tests
+* **revert**: To undo previous commits
+* **style**: Changes related to formatting or whitespace adjustments
+
+##### Subject line
```

Comment:
Use always "Title Case": https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case

Done for several other headings already in this file.